### PR TITLE
Update 'cn' locale 

### PR DESCRIPTION
**Pull request by KNA7022**
Translations to Simplified Chinese
All files should be correct

### CODE REVIEW

File: packages/client/locales/cn/account.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    "stopRoleplay": "停止角色扮演",
+    "addedToCall": "已添加到呼叫",
+    "statusUpdate": "状态更新",
+    "incomingCall": "来电",
+    "dotColor": "点阵颜色",
+    "fullRowColor": "整行颜色",
+    "userApiToken": "用户 API 令牌",
+    "reGenerateToken": "重新生成令牌",
+    "discord": "不和",
+    "steam": "蒸汽",
```
Comment (Dev-CasperTheGhost) [inline packages/client/locales/cn/account.json:43]: These are names of companies, these should stay the same.
```suggestion
    "discord": "Discord",
    "steam": "Steam",
```

File: packages/client/next.config.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 module.exports = {
   i18n: {
-    locales: ["en", "en_gb", "ru"],
+    locales: ["en", "en_gb", "cn","ru"],
```
Comment (Dev-CasperTheGhost) [inline packages/client/next.config.js]: just a small nit

```suggestion
    locales: ["en", "en_gb", "cn", "ru"],
```

**Review by Dev-CasperTheGhost — CHANGES_REQUESTED** [review]
2 minor nits

File: packages/client/locales/cn/account.json (hunk 1 above)
Comment (KNA7022) [reply packages/client/locales/cn/account.json:43]: ok I modified it and uploaded it

**Review by Dev-CasperTheGhost — APPROVED** [review]
looks good now!

**Pull request merged**